internal/xds/translator/testdata/out/xds-ir/listener-tcp-keepalive.clusters.yaml

```diff
@@ -66,3 +68,6 @@
   outlierDetection: {}
   perConnectionBufferLimitBytes: 32768
   type: EDS
+  upstreamConnectionOptions:
```

Comment:
the test is for listener side keep alive, how did this config come here (cluster) ?

Reply:
can you raise a follow up issue for this ?